lib/validator.js

```diff
@@ -97,37 +97,44 @@ async function _runValidator(
     return report;
   }
 
+  let result;
   try {
-    await validatorApi.validate(
+    result = await validatorApi.validate(
       {validatorInput, ledgerConfig, ledgerNode, validatorConfig});
   } catch(e) {
-    report.error = serializeError(e);
+    // if a validator throws, it is an uncaught exception. Serialize the error
+    // so that it will appear in the validator report for debugging purposes
+    result = {valid: false, error: serializeError(e)};
```

Comment:
Doesn't `serializeError` create a JSON object/string? Don't we want `error` to just be an `Error` instance?